test/faktory_worker_ruby_test.rb

```diff
@@ -0,0 +1,15 @@
+require_relative "test_helper"
+
+class FaktoryWorkerRubyTest < Minitest::Test
+  class FaktoryJob
+    include Faktory::Job
+    def perform(*_args); end
+  end
+
+  def test_read
```

Comment:
Can you add a test for the connect timeout as well (with `connect_url`) (or if you already tried, were there any issues)?

Reply:
Well it seemed this wasn't handled (yet). I'll give a ping if this PR is merged: https://github.com/contribsys/faktory_worker_ruby/pull/64